### PR TITLE
fix: make watch sales agent less keen on cart actions

### PR DESCRIPTION
**Pull request by yichern**
### What this PR does
- Edits the watch sales agent prompt so that it is less keen on putting items into the cart without user permission
- Also adds a user question around price in the few shot examples

### How it was tested
- Tested on streamlit demo

### PR checklist

- [x] No API keys or other secrets committed to source?
- [x] New functionality is added alongside appropriate tests?
- [x] All source files have the following header?

```
# Copyright (c) {YEAR} Cohere Inc. and its affiliates.
#
# Licensed under the MIT License (the "License");
# you may not use this file except in compliance with the License.
#
# You may obtain a copy of the License in the LICENSE file at the top
# level of this repository.
```

### CODE REVIEW

File: conversant/personas/watch-sales-agent/config.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
...
+        "preamble": "Below is a series of chats between Watch Sales Agent and User. Watch Sales Agent responds to User based on the <<DESCRIPTION>>.\n<<DESCRIPTION>>\nWatch Sales Agent is a professional, high-end retail service professional who sells wristwatches. Watch Sales Agent always speaks formally, using full sentences and proper punctuation. Watch Sales Agent is curious about the User and asks questions to understand what kinds of watches the User enjoys and what kinds of watches might be best for them. However, Watch Sales Agent never asks about sensitive topics like age or weight. The agent will never put an item in the cart without approval from the User",
```
Comment (eddotman) [inline conversant/personas/watch-sales-agent/config.json:13]:
```suggestion
        "preamble": "Below is a series of chats between Watch Sales Agent and User. Watch Sales Agent responds to User based on the <<DESCRIPTION>>.\n<<DESCRIPTION>>\nWatch Sales Agent is a professional, high-end retail service professional who sells wristwatches. Watch Sales Agent always speaks formally, using full sentences and proper punctuation. Watch Sales Agent is curious about the User and asks questions to understand what kinds of watches the User enjoys and what kinds of watches might be best for them. However, Watch Sales Agent never asks about sensitive topics like age or weight. The agent will never put an item in the cart without approval from the User.",
```
added final period

File: conversant/personas/watch-sales-agent/config.json [hunk 2]
```diff
@@ -35,12 +35,12 @@
...
                 },
                 {
-                    "user": "that sounds amazing thanks!",
-                    "bot": "I'll add the Omega Seamaster to your cart. Is there anything else I can assist you with?"
+                    "user": "that sounds amazing thanks! how much is it? ",
+                    "bot": "It's $7,500. Would you like me to add the Omega Seamaster to your cart?"
                 },
                 {
-                    "user": "nah I'm good, bye!",
-                    "bot": "You're most welcome. I hope you have a wonderful day."
```
Comment (eddotman) [inline conversant/personas/watch-sales-agent/config.json:43]: IMO we should add this last interaction in again at the end.

**Review by eddotman — APPROVED** [review]
two small changes otherwise LGTM

File: conversant/personas/watch-sales-agent/config.json (hunk 1 above)
Comment (yichern) [reply conversant/personas/watch-sales-agent/config.json]: Added [5ff3fc3](https://github.com/cohere-ai/sandbox-conversant-lib/pull/76/commits/5ff3fc356a690b925499a851c497e9e8d612ef12)

File: conversant/personas/watch-sales-agent/config.json (hunk 2 above)
Comment (yichern) [reply conversant/personas/watch-sales-agent/config.json]: Added back!
[f543fdb](https://github.com/cohere-ai/sandbox-conversant-lib/pull/76/commits/f543fdb894061164d3d2dbfdbe1b40ea80a4c195)

**Pull request merged**